codebundles/k8s-cortexmetrics-ingestor-health/sli.robot

```diff
@@ -0,0 +1,80 @@
+*** Settings ***
+Documentation       Uses kubectl to query the state of a ingestor ring and determine if it's healthy. Returns 1 if healthy, 0 if unhealthy.
+Metadata            Author    Shea Stewart
+
+Library             BuiltIn
+Library             RW.Core
+Library             RW.Utils
+Library             RW.K8s
+Library             RW.platform
+Library             OperatingSystem
+
+Suite Setup         Suite Initialization
+
+Force Tags          k8s    kubernetes    kube    k8    cortex    metrics    health
+
+
+*** Tasks ***
+Determine Cortex Ingester Ring Health
+    ${stdout}=    RW.K8s.Shell
+    ...    cmd=${binary_name} exec ${CORTEX_RESOURCE_NAME} -n ${NAMESPACE} --context ${CONTEXT} -it -- wget -O - --header 'Accept: application/json' http://127.0.0.1:8080/ring
```

Comment:
One thing to note is since 127.0.0.1 is hardcoded that you'll be limited to hitting the workload with exec that also has the ring endpoint on localhost, as opposed to if the user wants to target a "bastion pod" which is fine but you may want to include that in the docs.